versions.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+module-sets:
+  #Github Repo for Go Multimod: https://github.com/open-telemetry/opentelemetry-go-build-tools/tree/main/multimod
+  #Goals: Centralize all the module set into one files and use this version yaml to manage all the module set during the release.
+  adot-base:
+    version: v0.14.0
+    modules:
+      - github.com/aws-observability/aws-otel-collector
+      - github.com/aws-observability/aws-otel-collector/pkg/lambdacomponents
+  adot-tools:
+    version: v0.14.0
+    modules:
+      - github.com/aws-observability/aws-otel-collector/tools/linters
```

Comment:
Is this module expected to be versioned independently and to regularly change when the base modules do not?  I'm not sure I see a need to have a separate module set here.

Reply:
Same. At first I separate  the two since I intend to add another folder for the multi-mod but there is no reason now. Thanks for the notice.